src/fe-common/core/fe-common-core.c

```diff
@@ -462,25 +462,73 @@ void fe_common_core_finish_init(void)
 gboolean strarray_find_dest(char **array, const TEXT_DEST_REC *dest)
 {
 	g_return_val_if_fail(array != NULL, FALSE);
+	WI_ITEM_REC *item = window_item_find_window(dest->window, dest->server, dest->target);
```

Comment:
This line assumes that:

- `dest` is not `NULL`
- `dest->window` is not `NULL`
- `dest->target` is not `NULL`

I don't know right off the bat how likely is to end up in those situations but I'd add some checks to avoid a nasty crash.